cypress/integration/html-reader/navigations.ts

```diff
@@ -45,11 +55,10 @@ describe('navigating an EPUB page', () => {
       })
       .should('not.exist');
 
-    cy.wait(1000);
-
-    cy.findByRole('button', { name: '<' }).click();
+    cy.findByRole('button', { name: 'Previous Page' }).click();
+    cy.get('#reader-loading').should('be.visible');
 
-    cy.wait(1000);
+    cy.get('#reader-loading').should('not.be.visible');
```

Comment:
Yea I'm not totally sure, because I guess the iframe itself it loaded, but the reader is still loading the book content, and we are trying to wait on that completing, is that right? If you take this out, does it not retry the next failing test until it is in fact loaded?

Reply:
Testing library has a [WaitFor](https://testing-library.com/docs/dom-testing-library/api-async/#waitfor) - is that what you're looking for here?

Reply:
I think the `cy.get` is similar to WaitFor, where it also "re-try" until it finds the element or timeout reached, but somehow that didn't work.